itensor/iterativesolvers.h

```diff
@@ -1027,20 +1028,20 @@ applyExp(BigMatrixT const& A,
         Real t_step = std::min(t_out-t_now,t_new);
 
         auto V = std::vector<ITensor>(actual_maxiter+1);// storage for the bases of the Krylov subspace
-	auto H = CMatrix(actual_maxiter+2,actual_maxiter+2);// storage for the projected matrix
+        auto H = CMatrix(actual_maxiter+2,actual_maxiter+2);// storage for the projected matrix
 
-        V[0] = 1.0/beta * w;
+        V[0] = (w *= 1.0/beta);
```

Comment:
What was the reason for this change? I think the first version is fine (and more readable).